### PR TITLE
jc-1084-4 incorp peer edits for connector cli gs

### PR DESCRIPTION
**Pull request by MelissaFlinn**

### CODE REVIEW

File: docs/connectors/rhoas-cli-getting-started-connectors/README.adoc [hunk 1]
```diff
@@ -118,38 +119,38 @@ $ rhoas kafka create --name my-kafka-instance
...
 
-A Connectors namespace hosts your Connectors instances. 
+A {connectors} namespace hosts your {connectors} instances. 
 
-The namespace that you use depends on your OpenShift Dedicated environment.
+The namespace that you use depends on your OpenShift Dedicated (OSD) environment.
 
-* If you're using a trial cluster in your own OpenShift Dedicated environment:: The namespace is created when you add the {product-connectors} service to your trial cluster, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01[Adding and removing the Red Hat OpenShift Connectors add-on on your OpenShift Dedicated trial cluster^]. Your OSD trial cluster namespace is active for 60 days.
+If you're using a trial cluster in your own OSD environment:: The namespace is created when you add the {product-connectors} service to your trial cluster, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01[Adding the Red Hat OpenShift {connectors} add-on to your OpenShift Dedicated trial cluster^]. Your OSD trial cluster namespace is active for 60 days.
```
Comment (wtrocki) [inline docs/connectors/rhoas-cli-getting-started-connectors/README.adoc:151]: OSD might be not well known acronim

**Review by wtrocki — APPROVED** [review]

File: docs/connectors/rhoas-cli-getting-started-connectors/README.adoc (hunk 1 above)
Comment (MelissaFlinn) [reply docs/connectors/rhoas-cli-getting-started-connectors/README.adoc:151]: True - the first instance (see line 149) defines it OpenShift Dedicated (OSD)

Comment (bredamc) [reply docs/connectors/rhoas-cli-getting-started-connectors/README.adoc:151]: Actually, I need to retract one of my earlier comments. I think OSD is not an approved abbreviation, so we should not use it.

File: docs/connectors/rhoas-cli-getting-started-connectors/README.adoc [hunk 2]
```diff
@@ -387,13 +394,23 @@ To check the status:
...
+* `rhoas connector type -h` for viewing the available types of connectors
+* `rhoas connector list -h`` for listing {connectors} instances
+* `rhoas connector build -h` for building configuration files
+* `rhoas connector create -h` for creating {connectors} instances
 
 [role="_additional-resources"]
 .Additional resources
-* To access the `rhoas connector` help page, type `rhoas connector -h`
-{base-url-cli}{command-ref-url-cli}[_CLI command reference (rhoas)_^]
+* {base-url-cli}{command-ref-url-cli}[_CLI command reference (rhoas)_^]
```
Comment (bredamc) [inline docs/connectors/rhoas-cli-getting-started-connectors/README.adoc:413]: You can remove the bullet formatting again, now that you've added the new Commands section :)

**Review by bredamc — APPROVED** [review]
Just one teeny comment, so I'm approving :)

**Pull request merged**